<script>
    const user = async () => {
        try {
            await console.log("3:message")
            console.log("4:message")
            console.log("5:message")
        } catch (error) {
            console.log(error)
        }
    }
    user()
</script>